docsource/090_contribute.rst

```diff
@@ -10,6 +10,45 @@ In order to contribute to the OpenUpgrade project, please
   database, and give back any code improvements developed during the project.
 
 
+How to contribute new migration scripts
+---------------------------------------
+
+To Contribute to OpenUpgrade you must make one pull request per module.
+
+For example, if you want to propose the migration script of the ``account``
+module from version 15.0 to version 16.0:
+
+  * Always ensure that there is no work already in progress by a community member.
+    For that purpose, go the issue named "Migration to version 16.0" on github
+    `here <https://github.com/OCA/OpenUpgrade/issues/3681/>`_.
+
+  * Then, ensure that all the dependencies of the module are mark as ``done`` or
+    ``Nothing to do`` in OpenUpgrade for that version.
+    For that purpose, refer to :doc:`030_coverage_analysis`.
+    In our example, check the ``depends`` key of the `manifest <https://github.com/odoo/odoo/blob/16.0/addons/account/__manifest__.py#L18>`_ of the ``account`` module 
+    If some dependencies are missing, you should start by migrating these modules.
+
+  * As other OCA contribution, create a new branch, from an up to date OCA branch:
+
+    ``git checkout -b 16.0-mig-account``
+
+  * Make a copy of the analysis file.
+
+    ``cd ./openupgrade_scripts/scripts/account/16.0.1.2/``
+    ``cp upgrade_analysis.txt upgrade_analysis_work.txt``
+
+  * Explain the changes to do in the ``upgrade_analysis_work.txt``.
```

Comment:
Explain here about the format of the comments:

- `NOTHING TO DO`, and why it's nothing to do
- `DONE: pre-migration/post-migration:` indicating the what and maybe the why.
- `TODO:` This shouldn't be usually done.

Then, also explain that they can reorder lines and group them together for including all of them in a "logical" block for putting only one comment including all of them. Examples: all the noupdate=0 new and del ir* records, or the `NEW` and `DEL` lines for a field that is renamed.